.buildkite/scripts/run-bc-upgrade-tests.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the "Elastic License
+# 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+# Public License v 1"; you may not use this file except in compliance with, at
+# your election, the "Elastic License 2.0", the "GNU Affero General Public
+# License v3.0 only", or the "Server Side Public License, v 1".
+#
+
+set -euo pipefail
+
+echo "Selecting the most recent build from branch [$BUILDKITE_BRANCH]."
```

Comment:
I see these extra `echo`s, relative to #128614. Are these intentional?

Reply:
I used to diagnose the issue we were having, and found them useful for diagnostics/track progress so I left them there